content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+kubectl delete node <node name>
+```
+
+Lalu, pada simpul yang dihapus, atur ulang semua kondisi `kubeadm` yang telah dipasang:
+
+```bash
+kubeadm reset
+```
+
+Proses pengaturan ulang tidak mengatur ulang atau membersihkan kebijakan iptables atau tabel IPVS. Jika kamu ingin mengatur ulang iptables, kamu harus melakukannya secara manual:
+
+```bash
+iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
+```
+
+Jika kamu ingin mengatur ulang tabel IPVS, kamu harus menjalankan perintah berikut:
+
+```bash
+ipvsadm -C
+```
+
+Jika kamu ingin memulai kembali, cukup jalankan `kubeadm init` atau `kubeadm join` dengan
+argumen yang sesuai.
+
+### Membersihkan _control plane_
+
+Kamu dapat menggunakan `kubeadm reset` pada _host_ _control plane_ untuk memicu pembersihan
+best-effort.
+
+Lihat dokumentasi referensi [`kubeadm reset`](/docs/reference/setup-tools/kubeadm/kubeadm-reset/)
+untuk informasi lebih lanjut mengenai sub-perintah ini dan
+opsinya.
+
+{{% /capture %}}
+
+{{% capture discussion %}}
+
+## Selanjutnya {#selanjutnya}
+
+* Pastikan klaster berjalan dengan benar menggunakan [Sonobuoy](https://github.com/heptio/sonobuoy)
+* <a id="lifecycle" />Lihat [Memperbaharui klaster kubeadm](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
+  untuk detail mengenai pembaruan klaster menggunakan `kubeadm`.
+* Pelajari penggunaan `kubeadm` lebih lanjut pada [dokumentasi referensi kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm)
+* Pelajari lebih lanjut mengenai [konsep-konsep](/docs/concepts/) Kubernetes dan [`kubectl`](/docs/user-guide/kubectl-overview/).
+* Lihat halaman [Cluster Networking](/docs/concepts/cluster-administration/networking/) untuk daftar
+_add-on_ jaringan Pod yang lebih banyak.
+* <a id="other-addons" />Lihat [daftar _add-on_](/docs/concepts/cluster-administration/addons/) untuk
+  mengeksplor _add-on_ lainnya, termasuk perangkat untuk _logging_, _monitoring_, network policy, visualisasi &amp;
+  pengendalian klaster Kubernetes.
+* Atur bagaimana klaster mengelola _log_ untuk peristiwa-peristiwa klaster dan dari
+  aplikasi-aplikasi yang berjalan pada Pod.
+  Lihat [Arsitektur Logging](/docs/concepts/cluster-administration/logging/) untuk
+  gambaran umum tentang hal-hal yang terlibat.
+
+### Umpan balik {#feedback}
+
+* Untuk _bug_, kunjungi [kubeadm GitHub issue tracker](https://github.com/kubernetes/kubeadm/issues)
+* Untuk dukungan, kunjungi kanal Slack
+  [#kubeadm](https://kubernetes.slack.com/messages/kubeadm/)
+* Kanal Slack umum pengembangan SIG Cluster Lifecycle:
+  [#sig-cluster-lifecycle](https://kubernetes.slack.com/messages/sig-cluster-lifecycle/)
+* SIG Cluster Lifecycle [SIG information](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle#readme)
+* Milis SIG Cluster Lifecycle:
+  [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+
+## Kebijakan versi _skew_ {#version-skew-policy}
```

Comment:
```suggestion
## Kebijakan _version skew_ {#version-skew-policy}
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc